lib/tic_tac_toe_web_app.rb

```diff
@@ -0,0 +1,34 @@
+require 'sinatra/base'
+require 'board'
+require 'player_symbols'
+require 'one_indexed_grid_formatter'
+require 'web_display_to_board_adapter'
+require 'web_game'
+
+class TicTacToeWebApp < Sinatra::Base
+
+  set :views, File.dirname(__FILE__) + '/../views'
+  set :public_folder, File.dirname(__FILE__) + '/../public'
+
+  get '/' do
+    @formatted_rows = OneIndexedGridFormatter::new.format(Board.new)
+    @valid_moves = PlayerSymbols::all
+    erb :landing_page
+  end
+
+  get '/next_move' do
+    @move = params['move-taken']
+
+    contents_of_board = params['grid']
```

Comment:
Thanks Jim. Will do this update in a separate pull request. All the other comments I've addressed and added to this pr.